test/tests/utils/spline_interpolation/tests

```diff
@@ -1,8 +1,13 @@
 [Tests]
-  [./test]
+  [./regular_spline]
     type = 'Exodiff'
     input = 'spline_interpolation.i'
     exodiff = 'spline_interpolation_out.e'
   [../]
+  [./bicubic_spline]
+    type = 'Exodiff'
+    input = 'bicubic_spline_interpolation.i'
+    exodiff = 'bicubic_spline_interpolation_out.e'
+    custom_cmp = 'bicubic_spline_interpolation.cmp'
+  [../]
 []
-  
```

Comment:
I'm confused...in my text editor, I have a new-line at the end of this file, but it doesn't show up in the remote. And I don't have any trailing white-space in my editor. Sorry if I appear dense